Student/Student/Courses/Syllabus/StudentSyllabusViewController.swift

```diff
@@ -134,26 +134,24 @@ extension SyllabusViewController: UIScrollViewDelegate {
     }
 }
 
-extension SyllabusViewController: SyllabuseViewProtocol {
+extension StudentSyllabusViewController: StudentSyllabusViewProtocol {
     func updateNavBar(courseCode: String?, backgroundColor: UIColor?) {
         titleView.subtitle = courseCode
         navigationController?.navigationBar.useContextColor(backgroundColor)
         color = backgroundColor?.ensureContrast(against: .named(.white))
         menu.reload()
     }
 
-    func loadHtml(_ html: String?) {
-        guard let html = html else { return }
+    func updateMenuHeight() {
         menuHeightConstraint.constant = menuHeight
-        syllabus?.loadHTMLString(html, baseURL: nil)
```

Comment:
Where does the html get loaded now? Does it?

Reply:
There is now a `Core/SyllabusViewController`  that does the html.  `Core/SyllabusViewController is now a child of this ViewController.  I could change the name of this.  Any ideas on a better name for this viewController?

Reply:
Oh gotcha. This VC embeds the Core.SyllabusViewController. I missed that. I think it makes sense. The names are fine with me.